<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddAlarmVar</class>
 <widget class="QDialog" name="DialogAddAlarmVar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="comboBoxAlarmType"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="comboBoxVarName"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="comboBoxVarType"/>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>193</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddAlarmVar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddAlarmVar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
